hr_employee_document/i18n/fr.po

```diff
@@ -0,0 +1,38 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* hr_employee_document
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```

Comment:
I don't know if this could cause a translation error. Maybe new translations should be made or the file should be changed to a more updated one. As I said i don't know if it's an error or it doesn't matter. :)

Reply:
I don't sure translation file will be updated by oca-travis ? This below module just merged and oca-travis update automatically.
![Selection_480](https://user-images.githubusercontent.com/24691983/98136291-5d87d000-1ef3-11eb-9318-62cc0da14cd2.png)

Do you know about this ? @kittiu @Saran440

Reply:
@alex-guardado I think it doesn't matter. module will translate from .pot and bot will auto generate after merged.

Reply:
Okey, thank you for your answers. I didn't know how it worked exactly. Thank you :)